spinbot/policy/stale_issue_policy.py

```diff
@@ -8,34 +8,51 @@ def __init__(self):
         self.stale_days = self.config.get('stale_days')
         self.count = 0
         if not self.stale_days:
-            self.stale_days = 120
+            self.stale_days = 90
```

Comment:
Maybe make this 45? 90 * 3 (90 from last update until marked stale, 90 until marked to-be-closed, 90 until closed) seems like a long time.

Reply:
:+1: